rust/agama-dbus-server/src/questions.rs

```diff
@@ -224,6 +239,19 @@ impl Questions {
             }
         }
     }
+
+        /// tries to provide answer to question using answer strategies
```

Comment:
Describe what happens when multiple strategies do (not) provide an answer. Is there a short circuit? What if answers conflict?
I shouldn't need to read the implementation to know this.

Reply:
I will document it.